plugins/modules/redis_info.py

```diff
@@ -188,51 +208,33 @@
       "used_memory_startup": 791264
     }
 '''
-
-import traceback
-
-REDIS_IMP_ERR = None
-try:
-    from redis import StrictRedis
-    HAS_REDIS_PACKAGE = True
-except ImportError:
-    REDIS_IMP_ERR = traceback.format_exc()
-    HAS_REDIS_PACKAGE = False
-
-from ansible.module_utils.basic import AnsibleModule, missing_required_lib
-from ansible.module_utils.common.text.converters import to_native
-
-
-def redis_client(**client_params):
-    return StrictRedis(**client_params)
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.redis import (
+    fail_imports, redis_auth_argument_spec, RedisAnsible)
 
 
 # Module execution.
 def main():
-    module = AnsibleModule(
-        argument_spec=dict(
-            login_host=dict(type='str', default='localhost'),
-            login_port=dict(type='int', default=6379),
-            login_password=dict(type='str', no_log=True),
-        ),
-        supports_check_mode=True,
+    redis_auth_args = redis_auth_argument_spec(tls_default=False)
+    module_args = dict(
+        section=dict(choices=['all', 'clients', 'cluster', 'commandstats', 'cpu', 'default',
+                              'errorstats', 'everything', 'keyspace', 'latencystats', 'memory',
+                              'modules', 'persistence', 'replication', 'sentinel', 'server',
+                              'stats'],
+                     default='default', type='str')
     )
-
-    if not HAS_REDIS_PACKAGE:
-        module.fail_json(msg=missing_required_lib('redis'), exception=REDIS_IMP_ERR)
-
-    login_host = module.params['login_host']
-    login_port = module.params['login_port']
-    login_password = module.params['login_password']
+    module_args.update(redis_auth_args)
+    module = AnsibleModule(argument_spec=module_args, supports_check_mode=True)
+    fail_imports(module, module.params['tls'])
+    section = module.params['section']
 
     # Connect and check
-    client = redis_client(host=login_host, port=login_port, password=login_password)
+    client = RedisAnsible(module)
     try:
-        client.ping()
-    except Exception as e:
-        module.fail_json(msg="unable to connect to database: %s" % to_native(e), exception=traceback.format_exc())
-
-    info = client.info()
+        info = client.connection.info(section=section)
```

Comment:
Is `section='default'` the default value for `info()`? (Also has it been that for all supported versions of the redis library?)